tests/conftest.py

```diff
@@ -171,8 +171,13 @@ def bdb_port():
 
 
 @fixture
+def bdb_node_pubkey():
+    return environ['BIGCHAINDB_KEYPAIR_PUBLIC']
```

Comment:
Mmm, if someone runs this test locally, this value won't be correct. Let's just hardcode it for now.

Alternatively, we don't bother testing for the exact keypair in `test_node_info`.